Opc.Ua.CloudLib.Client/RestClient.cs

```diff
@@ -91,14 +91,14 @@ public async Task<List<UANodesetResult>> GetBasicNodesetInformation(List<string>
             return info;
         }
 
-        public async Task<AddressSpace> DownloadNodeset(string identifier)
+        public async Task<UANameSpace> DownloadNodesetAsync(string identifier)
```

Comment:
Nitpick: UANameSpace is not technically correct: a nodeset contains nodes from one (or more) namespaces. 
The class formerly known as AddressSpace is really a wrapper around a nodeset providing additional information about it. Or are there other uses envisioned for the future?
So, should we use something like UANodeSetInfo or UASNodeSetMeta?